cmd/kubeadm/app/constants/constants.go

```diff
@@ -237,10 +237,9 @@ var (
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
-		8:  "3.0.17",
 		9:  "3.1.10",
```

Comment:
We have changed etcd version to 3.1.11 for 1.9 (https://github.com/kubernetes/kubernetes/pull/57811). We should change the version here as well, right? But if so, it may cause issue when starting a 1.9.0 against HEAD kubeadm? @luxas @timothysc

Reply:
Yes.